### PR TITLE
Add list of tools and fix typos

### PR DESCRIPTION
**Pull request by victmo**
Guys please advise. Are these tools really usefull?
@AdiBurrack, do you want me to add more tools?

### CODE REVIEW

**Comment by adiburrack1** [comment]
@victmo Thanks! These tools are the best, no need for more tools for now

**Review by mingshao111 — COMMENTED** [review]
saw it.

**Review by adiburrack1 — APPROVED** [review]
approved

File: README.md [hunk 1]
```diff
@@ -5,12 +5,14 @@ Intro to Git and GitHub
 
 TODO
 
-## Expctatios
+## Expctations
```
Comment (victmo) [inline README.md:8]: Oh no! I see there's still a typo here.
@AdiBurrack will take care of this 🥇

**Review by mingshao111 — APPROVED** [review]

**Pull request merged**